templates/js/ai-assistant-bot/README.md.tpl

```diff
@@ -60,6 +62,15 @@ Before running or debugging your bot, please follow these steps to setup your ow
 ### Run Teams Bot locally
 
 1. First, select the Teams Toolkit icon on the left in the VS Code toolbar.
+{{#enableTestToolByDefault}}
+1. Press F5 to start debugging which launches your app in Teams App Test Tool using a web browser. Select `Debug in Test Tool`.
```

Comment:
Should we add preview tag in the readme as well?

Reply:
nice catch! will update the README as well.